### PR TITLE
enhancement(switch): Update the Switch to reflect the latest design

### LINKED ISSUE #858: Switch updates: add aria-hidden
After testing the react implementation of the Switch in JAWS, NVDA and VO, there's one minor change needed to improve the screen reader experience for JAWS and NVDA.

Where the following elements are included, they result in additional stops/keypresses for the screen reader user (in JAWS or NVDA):
`<span class="pf-c-switch__label pf-m-off">Off</span>`
`<span class="pf-c-switch__label pf-m-on">On</span>`

Having these labels announced doesn't provide any benefit to screen reader users, and is potentially confusing.

We can have these screen readers ignore these elements by added `aria-hidden="true"`:
`<span class="pf-c-switch__label pf-m-off" aria-hidden="true">Off</span>`
`<span class="pf-c-switch__label pf-m-on" aria-hidden="true">On</span>`

This will result in a single keypress to access the control, and when it receives focus, the control is announced with the aria-label given to the checkbox, and the state of the checkbox selection.

### LINKED ISSUE #875: Switch Component Enhancement
The current implementation of the Switch component does not conform to the latest design. The design was updated so that there should only be a label on one side of the switch. This is illustrated here:

[Switch.Animated.1.mov.zip](https://github.com/patternfly/patternfly-next/files/2531596/Switch.Animated.1.mov.zip)
 See discussion on #361

### PR DESCRIPTION
**Pull request by guillaumevincent**
Fixes #875
Fixes #858

### CODE REVIEW

**Comment by mcarrano** [comment]
This looks good @guillaumevincent . The only comment/question I have is that the label should change from On to Off when the switch is flipped to the left and vice versa. Not sure how you show that here without JS, but the way that the preview is now is somewhat misleading.

**Comment by guillaumevincent** [comment]
@mcarrano thank you for the merge
For me DOM modification should be done in JS with React

File: src/patternfly/components/Switch/switch.scss [hunk 1]
```diff
@@ -55,25 +52,16 @@
     }
 
     &:not(:checked) {
-      ~ .pf-c-switch__label.pf-m-off {
-        color: var(--pf-c-switch__input--not-checked__label--m-off--Color);
-      }
-
-      ~ .pf-c-switch__label.pf-m-on {
-        color: var(--pf-c-switch__input--checked__label--m-on--Color);
+      ~ .pf-c-switch__label {
```
Comment (mcoker) [inline src/patternfly/components/Switch/switch.scss]: I think you can take this CSS out of `&:not(:checked) {}` and just place it above `&:checked`?

**Comment by mcoker** [comment]
@mcarrano maybe the label should say something other than "on" or "off" for the example? It could just say "label" or "option" something generic like that.

**Comment by mcarrano** [comment]
Maybe that's a better idea @mcoker . If we can't toggle the actual label, then I think it's confusing as is. What do you think @guillaumevincent ?

File: src/patternfly/components/Switch/switch.scss (hunk 1 above)
Comment (guillaumevincent) [reply src/patternfly/components/Switch/switch.scss]: you're right

**Comment by guillaumevincent** [comment]
I agree @mcarrano 
Thank you @mcoker for the review

I updated the PR

**Pull request merged**